### PR TITLE
SUS-941: replace skin preview images

### PR DESCRIPTION
**Pull request by ghost**
JIRA case: https://wikia-inc.atlassian.net/browse/SUS-941
Code review: @sqreek

### CODE REVIEW

**Comment by macbre** [comment]
Let's care about the front-end performance and bandwidth usage :)

```
$ sudo apt-get install pngcrush

$ pngcrush -brute aliencrate_preview.png  aliencrate_preview_out.png

   Best pngcrush method        =   6 (ws 15 fm 5 zl 9 zs 0) =     8097
      for aliencrate_preview_out.png
     (6.30% critical chunk reduction)
     (7.23% filesize reduction)
```

File: extensions/wikia/ThemeDesigner/templates/ThemeDesigner_ThemeTab.php [hunk 1]
```diff
@@ -7,12 +7,12 @@
 		?>
 			<li data-theme="<?= $k ?>">
 				<label><?= ucfirst($k) ?></label>
-				<img src="<?= $wg->StylePath ?>/oasis/images/themes/<?= $k ?>_preview.jpg">
+				<img src="<?= $wg->StylePath ?>/oasis/images/themes/<?= $k ?>_preview.png">
```
Comment (macbre) [inline extensions/wikia/ThemeDesigner/templates/ThemeDesigner_ThemeTab.php]: Set `height` and `width` attributes.

**Comment by sqreek** [comment]
except macbre comment, LGTM

**Comment by macbre** [comment]
As discussed offline, let's try `pngquant` that will generate 8-bit PNG with differing applied.

**Pull request merged**